test/test_arguments.py

```diff
@@ -13,6 +13,7 @@
 
 import pytest
 
+import sockeye.constants as const
```

Comment:
we have a convention to import constants module as C (which is not ideal for code checking, I know, but we could keep it consistent here)